src/app/content/components/ContentExcerpt.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import { bodyCopyRegularStyle } from '../../components/Typography';
+import addTargetBlankToLinks from '../utils/addTargetBlankToLinks';
+
+interface Props {
+  content: string;
+}
+
+// tslint:disable-next-line:variable-name
+const ContentExcerpt = styled((props: Props) => {
+  const {
+    content,
+    ...excerptProps
+  } = props;
+
+  const fixedContent = React.useMemo(
+    () => addTargetBlankToLinks(props.content),
+    [props.content]
+  );
+
+  return <div
+    dangerouslySetInnerHTML={{ __html: fixedContent }}
+    {...excerptProps}
+  />;
+}).attrs({
+  className: 'content-excerpt',
```

Comment:
pretty sure this is going to break your ability to extend these styles, it would be more straightforward to do 

```
const ContentExcerpt = styled(({content, className, ...otherProps}: Props) => {
  ...
  return <div
    dangerouslySetInnerHTML={{ __html: fixedContent }}
    className={`content-excerpt ${className}`}
    {...otherProps}
  />;
})